server/index.js

```diff
@@ -64,17 +69,17 @@ router.get('/gamereviews/:gameid', async (req, res) => {
   }
 });
 
-// router.get('/reviewcount/:gameid', (req, res) => {
-//   fetch(`http://ec2-54-185-79-51.us-west-2.compute.amazonaws.com:3002/api/reviewcount/${req.params.gameid}`)
-//     .then(response => response.json())
-//     .then(results => {
-//       res.status(200).json(results);
-//     })
-//     .catch(e => {
-//       console.error(e);
-//       res.status(500).json({ error: 'Error fetching review counts' });
-//     });
-// });
+router.get('/reviewcount/:gameid', (req, res) => {
+  fetch(`http://ec2-54-185-79-51.us-west-2.compute.amazonaws.com:3002/api/reviewcount/${req.params.gameid}`)
+    .then(response => response.json())
```

Comment:
It might be worth checking with @GrumpyCat0709 to make sure this service is going to be available and verify that it is compatible - I think this was commented out by Christina because I took too long to provide this service for her, and she ended up generating the data herself. 

Theoretically, this data feed will be available, but by the looks of this legacy code, it might not be parsing correctly.

Check the App & Service plan vs this module's parsing routine -

A request like this:
http://[SERVER]:3001/api/reviews/[PRODUCT_ID] *

should return data like this:
/1: {"summary":"Mixed","percent":64,"positive":3750,"negative":2059,"total":5809}

Reply:
did you mean A request like this:

http://[SERVER]:3001/api/reviewscount/[PRODUCT_ID] * .. instead of reviews? 

but knowing that information helps a lot, I was playing with the front end and it seems to somehow still count the amount of reviews there is but when choosing the option of only seeing positive/negative reviews... it actually doesn't filter the reviews and shows me both positive/negative reviews regardless of parameters